__tests__/lib/openapi-to-json-schema.test.ts

```diff
@@ -908,14 +973,14 @@ describe('`example` / `examples` support', () => {
 
     it('should function through the normal workflow of retrieving a json schema and feeding it an initial example', () => {
       const operation = petstore.operation('/pet', 'post');
-      const schema: SchemaObject = operation.getParametersAsJSONSchema()[0].schema;
+      const schema = operation.getParametersAsJSONSchema()?.[0].schema as SchemaObject;
```

Comment:
isn't it better to avoid `as` here?

Reply:
It is but the types on all this JSON Schema and OpenAPI stuff is a complete disaster.